faith_domain/src/commonMain/kotlin/net/thechance/mena/faith/domain/repository/QuranRemoteDataSource.kt

```diff
@@ -0,0 +1,9 @@
+package net.thechance.mena.faith.domain.repository
+
+import net.thechance.mena.faith.domain.entity.Bookmark
+
+interface QuranRemoteDataSource {
```

Comment:
Should we make this `BookmarkRemoteDataSource` as it's currently only dealing with bookmarks?